Content.Client/Resources/ResourceTypes/ResourceSound.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.IO;
+using Robust.Client.Audio;
+using Robust.Client.Graphics;
+using Robust.Shared.Audio;
+using Robust.Shared.IoC;
+
+namespace Content.Client.Resources.ResourceTypes {
+    public class ResourceSound : DreamResource {
+        private readonly AudioStream _stream;
+
+        public ResourceSound(string resourcePath, byte[] data) : base(resourcePath, data) {
+            if (!resourcePath.EndsWith(".ogg")) {
```

Comment:
Clyde can load `.wav` too, FYI.

Reply:
I'm aware, but the original code didn't allow for it. Do I add support for it anyway?

Reply:
Added support for it, since it was only a few lines of code